Cabal/doc/developing-packages.rst

```diff
@@ -1,6 +1,9 @@
 Quickstart
 ==========
 
+.. TIP::
+    If this is your first time using `cabal` you should check out the `Getting Started guide <getting-started.html>`_.
```

Comment:
Nit: needs two trailing underscores (`__`).

Reply:
Sorry about that; I have done as you requested. Thank you.